lib/percy/client/snapshots.rb

```diff
@@ -14,6 +14,7 @@ def create_snapshot(build_id, resources, options = {})
             'attributes' => {
               'name' => options[:name],
               'enable-javascript' => options[:enable_javascript],
+              'enable-plain-rcw' => options[:enable_plain_rcw],
```

Comment:
enable on the basis of org. We only want to allow them for certain org. In the current way this will get enabled for everyone if they pass this attribute. If in some way we can check the org and pass the value here. May be check in `percy-api`

Reply:
yepp updating this in api to use only for test org